oas_docs/overlays/kibana.overlays.shared.yaml

```
@@ -61,6 +50,142 @@ actions:
         # description: 
         # url: https://www.elastic.co/docs/solutions/search/elastic-agent-builder
       x-displayName: 'Agent Builder'
+  - target: '$.tags[?(@.name=="Data streams")]'
+    description: Change tag description and displayName
+    update:
+      description: >
+        Data stream APIs enable you to manage data streams, which are collections of indices that share the same index template and are managed as a single unit for time-series data.
+      x-displayName: 'Data streams'
+  - target: '$.tags[?(@.name=="Elastic Agent actions")]'
+    description: Change tag description and displayName
+    update:
+      description: >
+        Elastic Agent actions APIs enable you to manage actions performed on Elastic Agents, including agent reassignment, diagnostics collection, enrollment management, upgrades, and bulk operations for agent lifecycle management.
+      x-displayName: 'Elastic Agent actions'
+  - target: '$.tags[?(@.name=="Elastic Agent binary download sources")]'
+    description: Change tag description and displayName
+    update:
+      description: >
+        Elastic Agent binary download sources APIs enable you to manage download sources for Elastic Agent binaries, including creating, updating, and deleting custom download sources for agent installations.
+      x-displayName: 'Elastic Agent binary download sources'
+  - target: '$.tags[?(@.name=="Elastic Agent policies")]'
+    description: Change tag description and displayName
+    update:
+      description: >
+        Elastic Agent policies APIs enable you to manage agent policies, including creating, updating, and deleting policies, as well as managing policy outputs, manifests, and auto-upgrade settings.
+      x-displayName: 'Elastic Agent policies'
+  - target: '$.tags[?(@.name=="Elastic Agent status")]'
+    description: Change tag description and displayName
+    update:
+      description: >
+        Elastic Agent status APIs enable you to retrieve status information about Elastic Agents, including health summaries and operational status.
```

Because there’s only one API endpoint under this namespace, maybe use the same structure as for the Fleet service tokens API:

```suggestion
        Enables you to retrieve status information about Elastic Agents, including health summaries and operational status.
```